less/index.less

```diff
@@ -32,28 +32,199 @@ footer, header, hgroup, menu, nav, section {
 body {
 	line-height: 1;
 }
+
 ol, ul {
 	list-style: none;
 }
+
 blockquote, q {
 	quotes: none;
 }
+
 blockquote:before, blockquote:after,
 q:before, q:after {
 	content: '';
 	content: none;
 }
+
 table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
+
 * {
     box-sizing: border-box;
+    font-family: 'Economica', sans-serif;
 }
 
-html, body {
-    height: 100%;
+html {
+//    font-size: 62.5%;
 }
 
+header {
+    display:flex;
+    justify-content:space-between;
+    margin: 10px;
+    font-family: sans-serif;
+    
+    h1 {
+        font-size: 30px;
+        font-family: 'Economica', sans-serif;
+    }
+    
+    nav {
+        display:flex;
+        flex-basis: 50%;
+        justify-content: space-between;
+        margin: 0 10px;
+        padding: 10px;
+        
+        a {
+            text-decoration: none;
+            color: black;
+        }
+    }
+}
+
+.container {
+    margin: 0 auto;
+    max-width: 1000px;
+    border: 1px solid black;
+}
+
+@baby-blue :  rgb(230, 243, 255);
+
+.top {
+    margin: 40px;
+    background: @baby-blue; 
+;
+    
+    img {
+        width:40%;
+        height:40%;
+        margin: 40px;
+    }
+    
+    h1 {
+        font-size: 32px;
+        margin: 20px;
+        border-bottom: 1px solid grey;
+        padding-bottom: 5px;
+    }
+    
+    p {
+        margin: 20px;
+    }
+    
+    button {
+        margin: 0 20px 20px 20px;
+        padding:10px;
+        background: grey;
+        color: white;
+    }
+}
+
+.main {
```

Comment:
I would recommend adding some padding to this `main` div to give your content a little more breathing room